auth/src/main/java/com/firebase/ui/auth/AuthUI.java

```diff
@@ -120,13 +120,15 @@
 
     /**
      * The set of authentication providers supported in Firebase Auth UI.
+     *
+     * TODO: the Github provider is not included because these tests do not
+     *       depend on the required auth-github module.
      */
     public static final Set<String> SUPPORTED_PROVIDERS =
             Collections.unmodifiableSet(new HashSet<>(Arrays.asList(
                     GoogleAuthProvider.PROVIDER_ID,
                     FacebookAuthProvider.PROVIDER_ID,
                     TwitterAuthProvider.PROVIDER_ID,
-                    GithubAuthProvider.PROVIDER_ID,
```

Comment:
Whoa, that's a public API, not part of `TestConstants`. 😛 I wasn't merged up with master before so I didn't see the changes. Here's how I fixed it:
https://github.com/firebase/FirebaseUI-Android/blob/98353a7164e4a55ca74538ea720069946193de28/auth/src/test/java/com/firebase/ui/auth/viewmodel/SocialProviderResponseHandlerTest.java#L74-L76

If we did that here:
https://github.com/firebase/FirebaseUI-Android/blob/98353a7164e4a55ca74538ea720069946193de28/auth/src/test/java/com/firebase/ui/auth/viewmodel/SocialProviderResponseHandlerTest.java#L319-L320
it should work.

Reply:
But yeah, I couldn't find a way to include GitHub in the tests. 😢